docs/book/src/developer/guide.md

```diff
@@ -27,7 +27,7 @@ You'll need the [docker daemon][docker] available.
 ### A Cluster
 
 You'll likely want an existing cluster as your [management cluster][mcluster].
-The easiest way to do this is with [kind], as explained in the [installation instructions][install].
+The easiest way to do this is with [kind], as explained in the quick start.
```

Comment:
Should this link to the quick start?

Reply:
we could, but it's really right there on the side